recipes-bsp/gpio-shutdown/files/gpio-shutdown-keymap.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
+
+# SPDX-FileCopyrightText: meta-raspberrypi contributors
```

Comment:
Let's drop this header for now. I'd like to clarify its contribution first.

Reply:
Done